functions/submission-created/submission-created.js

```diff
@@ -0,0 +1,37 @@
+const nodemailer = require('nodemailer')
+
+// // template inspired by https://css-tricks.com/using-netlify-forms-and-netlify-functions-to-build-an-email-sign-up-widget
+exports.handler = async event => {
+  const { EMAIL_ADDRESS, EMAIL_PASSWORD } = process.env
+  try {
+    const { name, email, message } = JSON.parse(event.body).payload.data
+
+    const transporter = nodemailer.createTransport({
+      service: 'Zoho',
+      host: 'smtp.zoho.com',
+      port: 465,
+      secure: true,
+      auth: {
+        type: 'SSL',
+        user: EMAIL_ADDRESS,
+        pass: EMAIL_PASSWORD,
+      },
+    })
+
+    const sendResult = await transporter.sendMail({
+      from: 'hello@bitsofgood.org',
```

Comment:
did you try changing this address to the email address provided in the contact form?

Reply:
I bet there's a good chance that messages will start going to spam if you do this. Perhaps add a Reply-To header instead?

Reply:
You can set the sender name though, e.g. `from: \`${name} <hello@bitsofgood.org>\`,` while also adding a `replyTo: email,`.

Reply:
@ArkaneMoose NodeMailer supports the `reply-to` header, as well as the `sender` header which exists for situations like a contact-us page. The spam filter should theoretically accept those two headers, since they both exist for the purposes of a contact-us page. We will have to see